contents/docs/integrations/android-integration.mdx

```diff
@@ -98,6 +98,50 @@ PostHog.with(this)
                                         .putValue("icon", "new2-final"));
 ```
 
+#### Setting user properties via an event
+
+To set properties on your users via an event, you can leverage the event properties `$set` and `$set_once`.
+
+##### $set
+
+**Example**
+
+```c
```

Comment:
should be `java`

Reply:
ah yes so this was mentioned elsewhere - `prism-react-renderer` doesn't support Java